formats/json/nativeMain/src/kotlinx/serialization/json/internal/FormatLanguage.kt

```diff
@@ -7,6 +7,16 @@ package kotlinx.serialization.json.internal;
 import kotlinx.serialization.InternalSerializationApi
 
 @InternalSerializationApi
+@Retention(AnnotationRetention.BINARY)
+@Target(
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY_GETTER,
+    AnnotationTarget.PROPERTY_SETTER,
+    AnnotationTarget.FIELD,
+    AnnotationTarget.VALUE_PARAMETER,
+    AnnotationTarget.LOCAL_VARIABLE,
+    AnnotationTarget.ANNOTATION_CLASS
+)
 public actual annotation class FormatLanguage(
```

Comment:
There's an `actual annotation class FormatLanguage` in JS source set, too

Also, what does the new rule says about `actual typealias`? We have this in JVM

Reply:
Fixed for Js source set.
"Annotations with retention SOURCE on expect are not checked if actual is typealias to compiled code." - and there is no compiler errors except error with FormatLanguage

Reply:
But `kotlin.Target` has default retention (runtime)